az-digital/az_quickstart
# Closes #1490: Adding content admin/editor permissions for node clone, block classes, metatag, reports

## Description
This PR adds several permissions to the content admin role:
- Administer block classes
- Administer Meta Tag
- View Site Reports
- Edit terms for page categories (I noticed this was missing when reviewing permissions)

It also add the following to the Arizona Quickstart Settings page that content admins can already access
- Set front page
- Set error pages (403/404)

The PR also gives permissions for content editors to clone pages (using Quick Node Clone).


## Related issues
#1490

## How to test
Test site: https://f9acc535-3b5b-46b0-aedf-a06ac37afc39.probo.build/

Log in as admin and make sure the following modules are enabled:
- Quick Node Clone
- Block Class
- Metatag

Log in as azcontentadmin:
- Verify that you can see Reports
- Verify that you can add a class to a block
- Verify that you can edit global Metatag settings
- Go to Arizona Quickstart Settings and verify you can set homepage, 404 page and 403 page

Log in as azcontenteditor:
- Verify that you can clone a page

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
